applications/asset_tracker/src/env_sensors/CMakeLists.txt

```diff
@@ -5,4 +5,31 @@
 #
 
 zephyr_include_directories(.)
-target_sources(app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/env_sensors.c)
+
+if (CONFIG_USE_BME680_BSEC)
+	if(CONFIG_FP_HARDABI)
+		target_sources(app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/bsec.c)
+		set (bsec_dir ${NRF_DIR}/ext/BSEC_1.4.7.4_Generic_Release)
+		set(BSEC_LIB_DIR     ${bsec_dir}/algo/Normal_version/bin/gcc/Cortex_M4F)
```

Comment:
Change Normal -> normal (to make it compile on linux, this probably compiles on win)

Reply:
done